sys/ztimer/Kconfig

```diff
@@ -0,0 +1,146 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+menuconfig MODULE_ZTIMER
+    bool "ztimer - High level timer abstraction layer"
+    depends on TEST_KCONFIG
+    select MODULE_ZTIMER_CORE
+    select MODULE_ZTIMER_CONVERT_FRAC
+    select MODULE_ZTIMER_CONVERT_SHIFT
+    select MODULE_ZTIMER_EXTEND
+
+if MODULE_ZTIMER
+
+menu "Backends"
+
+config MODULE_ZTIMER_PERIPH_RTC
+    bool "RTC peripheral"
+    depends on HAS_PERIPH_RTC
+    select MODULE_PERIPH_RTC
+
+config MODULE_ZTIMER_PERIPH_RTT
+    bool "RTT peripheral"
+    depends on HAS_PERIPH_RTT
+    select MODULE_PERIPH_RTT
+
+config MODULE_ZTIMER_PERIPH_TIMER
+    bool "Timer peripheral"
+    depends on HAS_PERIPH_TIMER
+    select MODULE_PERIPH_TIMER
+
+endmenu # Backends
+
+menu "Clocks"
+
+config MODULE_ZTIMER_USEC
+    bool "Microseconds"
+    depends on MODULE_ZTIMER_PERIPH_TIMER
+
+config MODULE_ZTIMER_MSEC
+    bool "Milliseconds"
+    depends on MODULE_ZTIMER_PERIPH_TIMER || MODULE_ZTIMER_PERIPH_RTT
```

Comment:
Shouldn't these also depend on `MODULE_ZTIMER`?

Reply:
They do, there is an `if` on line 16

Reply:
ah sorry for the noise